StickyTabBarViewController/Classes/ExpandableViewController.swift

```diff
@@ -131,40 +137,43 @@ internal class ExpandableViewController: UIViewController {
     }
     
     private func animateTransitionIfNeeded(isEnlarging: Bool, duration: TimeInterval) {
-        if runningAnimation == nil {
-            runningAnimation = UIViewPropertyAnimator(duration: duration,
-                                                      dampingRatio: 1) {
-                                                        if isEnlarging {
-                                                            self.heightConstraint.constant = self.deviceHeight - (self.tabController?.tabBar.frame.height ?? 0.0)
-                                                            self.minimisedView.alpha = 0.0
-                                                        } else {
-                                                            self.heightConstraint.constant = self.collapsedHeight
-                                                            self.minimisedView.alpha = 1.0
-                                                        }
-                                                        self.view.setNeedsLayout()
-                                                        self.tabController?.view.setNeedsLayout()
-                                                        self.view.layoutIfNeeded()
-                                                        self.tabController?.view.layoutIfNeeded()
-            }
-            
-            runningAnimation?.addCompletion { (position) in
-                switch position {
-                case .end:
-                    self.isEnlarged = !self.isEnlarged
-                default:
-                    ()
+        guard
+            runningAnimation == nil,
+            let tabController = tabController else {
+                return
+        }
+        
+        runningAnimation = UIViewPropertyAnimator(
```

Comment:
I like to have enabled the column limit in Xcode so it does not go beyond the limit Github sets so it's easier to red for reviewers. 

This is totally a personal preference (highly influenced by Line-Wrapping followed by Google https://google.github.io/swift/). But of course this is up for discussion. 
![Screenshot 2020-04-09 at 20 48 25](https://user-images.githubusercontent.com/6764091/78929921-7fc38780-7aa3-11ea-9402-63e1dca5e86b.png)

Reply:
Yes thats a great idea. Would you like it to be 100 or 120, on google docs it is mentioned as 100.

Reply:
I think 100 it's a bit tight & I believe 120 works well on Github. We can start with 120 & change it when we feel like it 👍